### DISCUSSION TITLE
[feature request] Sort while searching for repos on mobile app.

### DISCUSSION THREAD

**Discussion by nadir500** [community/community#3599] [Mobile]
Sort with least stars, date updated, oldest repos etc...

**Comment by PranamBhat** [comment]
This feature is available.

Kindly search for a repository name and do the sorting like this:

![git](https://user-images.githubusercontent.com/32120033/112434900-0a130300-8d6a-11eb-9977-96dc30f319f8.JPG)

**Reply by nadir500** [reply]
I was talking about the mobile version, I'll update the post.
